src/components/Guitar.tsx

```
@@ -0,0 +1,156 @@
+  newSynth.set({
+    oscillator: {
+      type: 'fmsine',
+    },
+    envelope: {
+      attack: 0.01,
+      decay: 0.2,
+      sustain: 0.5,
+      release: 1.5,
+    },
+  })
+  const [synth, setSynth] = useState<Tone.PolySynth<Tone.Synth<Tone.SynthOptions>> | null>(newSynth)
```

ESLint

[error] 'setSynth' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)